colosseum/units.py

```diff
@@ -19,7 +19,10 @@ def lu(self, display=None, font=None, size=None):
         return round(LU_PER_PIXEL * self.val)
 
     def px(self, display=None, font=None, size=None):
-        return self.lu(display=display, font=font, size=size) // LU_PER_PIXEL
+        logical_units = self.lu(display=display, font=font, size=size)
+        value = logical_units / LU_PER_PIXEL
+        int_value = int(value)
+        return int_value if value == int_value else value
```

Comment:
Was there a reason for this to return ints @freakboy3742. Some of the tests expect float pixel values.

Reply:
The primary reason was that the rendering platforms we're targetting (i.e., widget toolkits) don't allow float positioning. Which test was expecting float values?

Reply:
All the test that now pass

Reply:
>don't allow float positioning

That should probably be handled at the toga level, not the colosseum level ?

Reply:
You're correct - it should be. Not sure how I missed that detail originally.